### ISSUE TITLE
functionality altered when using Screen Sharing to a remote Mac

### ISSUE THREAD

**Issue by mummifiedclown** [Clipy/Clipy#547]
On the remote Mac, the normal Cmd-Shift-v only brings up the History; Cmd-Shift-f must be used to get to Snippets.  Also, if any item but the most recent in the History is selected, just a text "v" is entered.  To get that History entry, it has to be selected again after it moves to the top, then another "v" is entered and only after that can the item be pasted.

**Comment by sunnyatnutanix** [comment]
Do you have a workaround on this, have the same issue an v input automatically. I am trying to debug the issue and see if it can be fixed.do you have any suggestions thx.

**Comment by mummifiedclown** [comment]
@sunnyatnutanix No, haven't come up with anything yet, other than going back to a lot of back and forth cutting and pasting.  Only seems to be a problem with vnc/Apple Remote Desktop.  TeamViewer doesn't have the problem, but it has its own issues which is why I avoid it for WFH situations.  Still really handy for the snippets though.

**Comment by sunnyatnutanix** [comment]
@mummifiedclown I think I figured out, with v input when using remote assist apple, like rdp, zoom, etc. you need to disable cmd+v as input, then v will not be input into the remote client. let me know if it works for you.

**Comment by mummifiedclown** [comment]
@sunnyatnutanix That actually didn't work for me - if I turn off that setting, Clipy doesn't work at all.
However, I did figure out what my problem was: I'm also running Clipy on my local Mac.  If I quit it locally, Clipy on the remote Mac functions normally.  So Issue is Clipy snafu'ing itself over the shared clipboard...
